packages/in_app_purchase/lib/src/store_kit_wrappers/sk_payment_queue_wrapper.dart

```diff
@@ -103,9 +103,11 @@ class SKPaymentQueueWrapper {
   /// finishTransaction:]`](https://developer.apple.com/documentation/storekit/skpaymentqueue/1506003-finishtransaction?language=objc).
   Future<void> finishTransaction(
       SKPaymentTransactionWrapper transaction) async {
+    Map requestMap = transaction.toFinishMap();
```

Comment:
Prefer lightweight generic. Consider adding a type for the `Map`. For example: Map<String, String>

Reply:
Done